src/components/MetricsOptions/MetricsOptionsBar.tsx

```diff
@@ -152,7 +173,21 @@ export class MetricsOptionsBar extends React.Component<Props, MetricsOptionsStat
               </MenuItem>
             ))}
           </DropdownButton>
+          <DropdownButton id="pollInterval" title="Polling interval" onSelect={this.onPollIntervalChanged}>
+            {MetricsOptionsBar.PollIntervals.map(r => (
+              <MenuItem key={r[0]} active={r[0] === this.state.pollInterval} eventKey={r[0]}>
+                {r[1]}
+              </MenuItem>
+            ))}
+          </DropdownButton>
         </div>
+        <ToolbarRightContent>
+          {this.props.loading && (
+            <span>
+              <Icon name="spinner" spin={true} size="lg" /> Loading
```

Comment:
It looks different from the patternfly spinner, no? Is it intentional?

Reply:
Yes, render the icon spinner consume less memory. But for me there is not problem if you prefer Spimmer

Reply:
It's just a matter of being consistent with the other places we have a spinner. I haven't seen if they have been changed in the other places, but I think either we change it everywhere, or nowhere.

Reply:
PS: maybe if the PF spinner is an issue it should be addressed globally through another task

Reply:
mmm I agree with you, Should I keep Spinner while we discuss this with others?